docs/redirects.json

```diff
@@ -0,0 +1,104 @@
+{
+  "redirects": [
+    {
+      "source": "/docs/solana-playground",
+      "destination": "/docs/quickstart/solpg"
+    },
+    {
+      "source": "/docs/hello-world",
+      "destination": "/docs/quickstart/local"
+    },
+    {
+      "source": "/docs/intro-to-solana",
+      "destination": "/docs"
+    },
+    {
+      "source": "/release-notes/0.30.1",
+      "destination": "/docs/updates/release-notes/0-30-1"
+    },
+    {
+      "source": "/release-notes/0.30.0",
+      "destination": "/docs/updates/release-notes/0-30-0"
+    },
+    {
+      "source": "/release-notes/0.29.0",
+      "destination": "/docs/updates/release-notes/0-29-1"
+    },
+    {
+      "source": "/release-notes/changelog",
+      "destination": "/docs/updates/changelog"
+    },
+    {
+      "source": "/docs/high-level-overview",
+      "destination": "/docs/basics/program-structure"
+    },
+    {
+      "source": "/docs/the-accounts-struct",
+      "destination": "/docs/basics/program-structure"
+    },
+    {
+      "source": "/docs/the-program-module",
+      "destination": "/docs/basics/program-structure"
+    },
+    {
+      "source": "/docs/errors",
+      "destination": "/docs/features/errors"
+    },
+    {
+      "source": "/docs/cross-program-invocations",
+      "destination": "/docs/basics/cpi"
+    },
+    {
+      "source": "/docs/pdas",
+      "destination": "/docs/basics/pda"
+    },
+    {
+      "source": "/docs/contribution-guide",
+      "destination": "/docs"
```

Comment:
If we're removing this, I think we should document this in GitHub somewhere (as an issue template?).

Reply:
added back